service_report/site-within-lpa-validator.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: This notebook makes use of a polygon of england when determining where a brownfield site is located. This polygon can be downloaded here: https://cartographyvectors.com/map/1321-england-uk, by selecting 'Choose Map Format' and then 'Well-Known-Text (WKT)'. The polygon is then stored in the same directory as this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "england_poly_json = \"englandjson.wkt\"\n",
```

Comment:
Can you consider keeping this file in the folder instead of the user downloading it?